src/hotspot/share/code/oopRecorder.cpp

```diff
@@ -68,10 +68,10 @@ template <class T> void ValueRecorder<T>::copy_values_to(nmethod* nm) {
 template <class T> void ValueRecorder<T>::maybe_initialize() {
   if (_handles == nullptr) {
     if (_arena != nullptr) {
-      _handles  = new(_arena) GrowableArray<T>(_arena, 10, 0, 0);
+      _handles  = new(_arena) GrowableArray<T>(_arena, 10, 0, T{});
       _no_finds = new(_arena) GrowableArray<int>(    _arena, 10, 0, 0);
```

Comment:
Can you remove spaces before `_arena` here and in following block since you are changing code near.

Reply:
done.